api/CSS.json

```diff
@@ -477,43 +477,6 @@
           }
         }
       },
-      "ic": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSS/ic",
-          "spec_url": "https://drafts.css-houdini.org/css-typed-om/#dom-css-ic",
-          "support": {
-            "chrome": {
-              "version_added": false,
-              "notes": "See <a href='https://crbug.com/937101'>bug 937101</a>."
```

Comment:
This bug was resolved but the feature has not been enabled and is not available in the latest canary release.